src/main/java/us/kbase/sdk/validator/ModuleValidator.java

```diff
@@ -131,9 +144,19 @@ public int validate() {
 				continue;
 			}
 			
+			String methodStoreUrl = null;
+			try {
+				methodStoreUrl = getMethodStoreUrl(module.toPath(), kyc);
+			} catch (IOException | IllegalStateException e) {
+				System.err.println("  **ERROR** - getting Narrative Method Store URL failed:");
+				System.err.println("                "+e.getMessage());
```

Comment:
no spacing around the `+`?

Reply:
Good god. Fixed this and a few other places that were the source of this copy and paste atrochity